components-core/src/main/java/com/adyen/checkout/components/base/BaseConfigurationBuilder.kt

```diff
@@ -23,6 +23,8 @@ abstract class BaseConfigurationBuilder<
     protected var clientKey: String
 ) {
 
+    protected var isAnalyticsEnabled: Boolean? = null
```

Comment:
A nullable boolean feels a bit wrong to me 🤔 WDYT?

Reply:
I agree it looks weird but so does a nullable string or enum that will never actually be null when it reaches the component. It's just because we decided setting all the default values in the component params mappers and not in the configuration builders. I did that as well in other configurations. WDYT?

Reply:
Okay let's keep it nullable 👍